Okta/OktaAuth/OktaTokenManager.swift

```diff
@@ -68,17 +68,25 @@ open class OktaTokenManager: NSObject, NSCoding {
     }
 
     required public convenience init?(coder decoder: NSCoder) {
-        try? self.init(
-                    authState: decoder.decodeObject(forKey: "authState") as! OIDAuthState,
-                       config: decoder.decodeObject(forKey: "config") as! [String: String],
-                accessibility: (decoder.decodeObject(forKey: "accessibility") as! CFString)
+        guard let configDict = decoder.decodeObject(forKey: "config") as? [String : String] else {
+            return nil
+        }
+        
+        guard let config = try? OktaAuthConfig(with: configDict) else {
+            return nil
+        }
+        
+        self.init(
+            authState: decoder.decodeObject(forKey: "authState") as! OIDAuthState,
```

Comment:
I think more safe would be to decode and then check whether we can successfully cast to OIDAuthState. Just to cover corner cases with corrupted data streams and model integrity issues

Reply:
Done.